.travis.yml

```
@@ -65,7 +65,7 @@ deploy:
       mvn deploy -DskipTests --settings ci-tools/common/maven-settings.xml
     skip_cleanup: true
     on:
-      branch: KTO-244
+      branch: KTO-254
```

```suggestion
      branch: master
```
Varmaan turha muistutus, mutta varmuudeksi.